api-reference/_hidden/GridBaseColumn/dataField.md

```diff
@@ -55,5 +58,15 @@ The **columns** array can contain only the names of data source fields, which is
     
 ---
 
+[note]
+
+Please review the following notes about data binding:
+
+- If you create an unbound column (use the [calculateCellValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#calculateCellValue) function), specify the **columns[]**.[name](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#name) property instead.
```

Comment:
```suggestion
- If you create an unbound column (use the [calculateCellValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#calculateCellValue) function), specify the **columns[]**.[name](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#name) property instead of **dataField**.
```